spec/traits.dd

```diff
@@ -411,6 +413,33 @@ static assert(!__traits(isTemplate,foo!int()));
 static assert(!__traits(isTemplate,"string"));
 ---
 
+$(H2 $(GNAME isModule))
+
+        $(P Takes one argument. If that argument is a symbol that refers to a module then
+        $(D true) is returned, otherwise $(D false). Packages that contain a `package.d`
+        file are also considered to be modules.
+        )
+
+---
+import std.algorithm.sorting;
+static assert(!__traits(isModule, std));
+static assert(__traits(isModule, std.algorithm));
+static assert(__traits(isModule, std.algorithm.sorting));
+---
+
+$(H2 $(GNAME isPackage))
+
+        $(P Takes one argument. If that argument is a symbol that refers to a package
+        then $(D true) is returned, otherwise $(D false).
```

Comment:
I would add another sentence here  - defining what a package is. Maybe sth. like this?

>  A package is a grouping of modules. `package.d` modules are considered as a package, however the existence of a `package.d` file for a package isn't required.